sdk/core/azure-core/azure/core/pipeline/policies/_redirect.py

```diff
@@ -45,14 +45,14 @@ class RedirectPolicyBase(object):
 
     REDIRECT_STATUSES = frozenset([300, 301, 302, 303, 307, 308])
 
-    REDIRECT_HEADERS_BLACKLIST = frozenset(['Authorization'])
+    REDIRECT_HEADERS_BLOCKLIST = frozenset(['Authorization'])
```

Comment:
@xiangyan99 is this going to cause any issues if we change the name here?

Reply:
I think this is acceptable (from a breaking change perspective). And the right thing to do.